tools/validators/ontology_validator/yamlformat/tests/entity_type_manager_test.py

```diff
@@ -249,5 +249,83 @@ def testAnalyzeHandlesNamespaces(self):
     self.assertTrue(parent.HasFindingTypes(
         [findings_lib.DuplicateExpandedFieldSetsWarning]))
 
+  def testGetCompleteFieldSetsOI(self):
+    yaml = {'literals': ['field1', 'field2', 'field3', 'field4']}
```

Comment:
for both these tests you can create either a simpler data set or have an ontology with no types to allow you to positively test the output.  (if the input or is simple or empty, you should be able to manually construct the correct output to match).